### PR TITLE
Meshfill animation

### PR DESCRIPTION
**Pull request by doutriaux1**
@aashish24 @dlonie goes with: UV-CDAT/uvcdat-testdata#44

### CODE REVIEW

**Comment by doutriaux1** [comment]
@chaosphere2112 you to

**Comment by dlonie** [comment]
Tests are failing: https://open.cdash.org/viewTest.php?onlyfailed&buildid=3794735

Also, I'm having trouble figuring out the new variable name:  doCellPointForScalar. What does this mean? It's the 'CellPoint' bit that throwing me off.

File: Packages/vcs/Lib/VTKPlots.py [hunk 1]
```diff
@@ -1032,6 +1032,10 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
               mapper.SetScalarModeToUseCellData()
               mappers.append([mapper,])
           else:
+              if isinstance(gm,vcs.meshfill.Gfm):
+                  doCellPointForScalar = True
+              else:
+                  doCellPointForScalr = False
```
Comment (dlonie) [inline Packages/vcs/Lib/VTKPlots.py]: Scalar

**Comment by chaosphere2112** [comment]
Works for me, passes tests: https://open.cdash.org/viewTest.php?onlypassed&buildid=3795126

**Comment by aashish24** [comment]
Would be nice to rename the variable to something better. Cell point is confusing..

**Comment by doutriaux1** [comment]
@aashish24 the var actually says what it does, if set to True then when running Update for animation we need to do SetScalarRangeCellPoint so doCellPoint seemed like a good name. But feel free to edit and give a better name. Keep in mind with @dlonie updated workflow I suspect most of this will disappear...

**Comment by aashish24** [comment]
sure. @doutriaux1. Let's get it merged and we will clean it up if necessary with @dlonie's branch.

**Pull request merged**